blog/BCHN-Financial-Report-2020-10-20.md

```diff
@@ -0,0 +1,218 @@
+
+        Main file                : bchn.journal
+        Included files           : prnc2exht3zxlrqqcat690tc85cvfuypngh7szx6mk.journal
+                                   pp03e95qz7yene7v4vdx2vfsqx4qsq4wfvw8z3pkjf.journal
+        Transactions span        : 2020-02-22 to 2020-10-14 (235 days)
+        Last transaction         : 2020-10-13 (7 days ago)
+        Transactions             : 207 (0.9 per day)
+        Transactions last 30 days: 17 (0.6 per day)
+        Transactions last 7 days : 1 (0.1 per day)
+        Payees/descriptions      : 1
+        Accounts                 : 20 (depth 4)
+        Commodities              : 2 (, BCH)
+
+
+### Notes about balances displayed on block explorers
+
+Block explorers may show a slightly lower amount for the main wallet address `prnc2exht3zxlrqqcat690tc85cvfuypngh7szx6mk` (2,109.93763055 BCH at block 657,989).
+
+This is because a small amount in donations (currently 0.00075823 BCH) are stored on a subsequent address (`pp03e95qz7yene7v4vdx2vfsqx4qsq4wfvw8z3pkjf`) in BCHN's multi-signature wallet which was issued to Coin.dance who are collecting donations on behalf of node client projects. They requested this separate address for their own accounting purposes. BCHN will probably periodically receive some donations coming in from there - should significant amounts accrue, they will eventually be moved to the main wallet, but the report (as above) will reflect the total donated amounts on both our main and this secondary address.
+
+
+### SLP donation address
+
+The project has a SimpleLedger (SLP) token donation address.
+
+`simpleledger:qrjv4klcn9z5kqeye3ps64pr6aa4mzux8cy776pm6k`
+
+Currently, the project has received and is holding donations of 2,000,000 SPICE 🌶️ and 1,000 USDt on that address.
+
+A few other unknown tokens have been sent to the address.
+
+As we conduct our payments exclusively in Bitcoin Cash up to now, we have yet to set up a ledger file to account for token holdings. We may do so for main tokens like SPICE, but not generally for all random tokens sent our way, as this would be a way to spam the project.
+
+
+### Development bounties
+
+The project has had good experience with bountied work so far and has increased that share of its expenditure.
+
+In this last period, bounties were used to successfully source
+
+* website content (importing from read.cash to the Newsroom)
+
+* development of a relatively complex new functional test related to ongoing performance improvements
+
+* development of a new `indexdir` feature (complete with implementation, development and documentation) requested by a user
+
+Developers who are interested in partaking in upcoming bounties, are invited to join the project Slack (link below) for announcements in the development channels and keep an eye out for [Issues on GitLab marked with a `bounty` label](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=bounty).
```

Comment:
also can we add link to github?

Reply:
The BCHN GitHub repo is for releases only I believe, it doesn't have an "issue" tab.